Makefile

```diff
@@ -124,12 +124,12 @@ sdk: .bin/swagger .bin/ory node_modules
 quickstart:
 		docker pull oryd/kratos:latest
 		docker pull oryd/kratos-selfservice-ui-node:latest
-		quickstart -f quickstart.yml -f quickstart-standalone.yml up --build --force-recreate
+		docker-compose -f quickstart.yml -f quickstart-standalone.yml up --build --force-recreate
 
 .PHONY: quickstart-dev
 quickstart-dev:
-		docker build -f .docker/Dockerfile-build -t oryd/kratos:latest .
-		quickstart -f quickstart.yml -f quickstart-standalone.yml -f quickstart-latest.yml $(QUICKSTART_OPTIONS) up --build --force-recreate
+#		docker build -f .docker/Dockerfile-build -t oryd/kratos:latest .
```

Comment:
```suggestion
		docker build -f .docker/Dockerfile-build -t oryd/kratos:latest .
```

Reply:
yeah about that is there a quickstart binary ?

Reply:
revert